lightning-network-book.tex

```diff
@@ -713,12 +713,24 @@ \subsection{Denial of Service attacks}
 \item Spam HTLCs to block channels
 \item Spam the blockchain
 \end{itemize}
+\subsection{Loss of keys or passwords}
+\todo{E.g. \url{https://arstechnica.com/information-technology/2019/02/digital-exchange-loses-137-million-as-founder-takes-passwords-to-the-grave/}}
 \chapter{Controversial topics}
 \section{Consequences of decentralization}
 \section{Custodial Services, Lightning hubs and banks}
 \section{Privacy and regulatory challenges}
 \section{Consensus, Forking and Altcoins}
-
+\chapter{The near and long-term future of Lightning}
```

Comment:
while I like the topics of this chapter I am not sure if they should be summarized in this chapter instead of being spread in other chapters

Reply:
Just a remark: the reader should clearly understand that the Lightning ecosystem is just being built up and that there is a lot that lies in the future. Also, some people might NOT read the whole book. They will jus tbrowse through the TOC and then read the chapters that interest them. For SUCH people a separate chapter might be more useful than working it into various other chapters throughout the book.